packages/caliper-fabric/test/connector-versions/peer-gateway/PeerGateway.js

```diff
@@ -0,0 +1,440 @@
+        await fabricGateway.init().should.not.be.rejected;
+    });
+
+    it('should be able to initialise in preperation for use by a caliper manager when mutual tls is false', async () => {
+        const connectorConfiguration = await new ConnectorConfigurationFactory().create(path.resolve(__dirname, configWith2Orgs1AdminInWalletNotMutual), walletFacadeFactory);
+        const fabricGateway = new PeerGateway(connectorConfiguration, 1, 'fabric');
+        await fabricGateway.init().should.not.be.rejected;
+    });
+
+    it('should do nothing when attempting to install a smart contract', async () => {
+        const connectorConfiguration = await new ConnectorConfigurationFactory().create(path.resolve(__dirname, configWith2Orgs1AdminInWalletNotMutual), walletFacadeFactory);
+        const fabricGateway = new PeerGateway(connectorConfiguration, 1, 'fabric');
+        await fabricGateway.installSmartContract().should.not.be.rejected;
+    });
+
+    it('should throw an error and not return a context when preparing for use by a caliper worker when mutual tls is enabled (true)', async () => {
+        const connectorConfiguration = await new ConnectorConfigurationFactory().create(path.resolve(__dirname, configWith2Orgs1AdminInWallet), walletFacadeFactory);
+        const fabricGateway = new PeerGateway(connectorConfiguration, 1, 'fabric');
+        await fabricGateway.getContext().should.be.rejectedWith('Mutual tls is not supported with the Peer Gateway Connector');
+    });
+
+    it('should return a context when preparing for use by a caliper worker', async () => {
+        const connectorConfiguration = await new ConnectorConfigurationFactory().create(path.resolve(__dirname, configWith2Orgs1AdminInWalletNotMutual), walletFacadeFactory);
+        const fabricGateway = new PeerGateway(connectorConfiguration, 1, 'fabric');
+        const context = await fabricGateway.getContext();
+        context.should.be.instanceOf(FabricConnectorContext);
+    });
+
+    it('should return the same context when requested multiple times', async () => {
+        const connectorConfiguration = await new ConnectorConfigurationFactory().create(path.resolve(__dirname, configWith2Orgs1AdminInWalletNotMutual), walletFacadeFactory);
+        const fabricGateway = new PeerGateway(connectorConfiguration, 1, 'fabric');
+        const context = await fabricGateway.getContext();
+        const context2 = await fabricGateway.getContext();
+        context2.should.equal(context);
+    });
+
+    it('should create one Client for each organization', async () => {
+        const connectorConfiguration = await new ConnectorConfigurationFactory().create(path.resolve(__dirname, configWith2Orgs1AdminInWalletNotMutual), walletFacadeFactory);
+        const fabricGateway = new PeerGateway(connectorConfiguration, 1, 'fabric');
+        await fabricGateway.getContext();
+        GrpcClient.options[0]['grpc.ssl_target_name_override'].should.deep.equal('peer0.org1.example.com');
```

Comment:
For this test you could just do
```
GrpcClient.options.length.should.equal(2);
```
however I think for readability you should replicate what was done with the gateway and track construction so you could do something like
```
GrpcClient.constructed.should.equal(4);
```